.github/workflows/build_runson_ami.yaml

```diff
@@ -0,0 +1,52 @@
+name: Build RunsOn AMI
+
+on:
+    workflow_dispatch:
+    schedule:
+      # Runs at 12:00 PM (noon) on the 1st day of every month
+      - cron: '0 12 1 * *'
+
+jobs:
+  build_ami:
+    name: Build-AMI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          # TODO: Rename this PAT to something that encompasses more than just cherry-picking
+          # but all PR creation (such as this one).
+          token: ${{ secrets.WORKER_PANTS_CHERRY_PICK_PAT }}
+      - name: Run packer
+        env:
+            GH_TOKEN: ${{ secrets.WORKER_PANTS_CHERRY_PICK_PAT }}
```

Comment:
Can you just use the existing `GITHUB_TOKEN` with enhanced permissions via a `permissions:` config in this workflow file?

Reply:
Example: https://github.com/tdyas/pyenv-python-build-standalone/blob/c01a1ec31203a5f8badbf76525771dee1e4e66b2/.github/workflows/check-for-new-releases.yaml#L9-L11

Reply:
I don't know, I am just cargo-culting the token stuff. I am happy to go with a different cult...

Reply:
Ah yes, your pyenv scraper seems like a better thing for me to crib off...